app/letsencrypt_service

```diff
@@ -83,7 +83,7 @@ update_certs() {
             acme_ca_uri="https://acme-staging.api.letsencrypt.org/directory"
             if [[ ! -f /etc/nginx/certs/.${base_domain}.test ]]; then
                 # Remove old certificates
-                rm -rf /etc/nginx/certs/${base_domain}
+                [[ ! -z "${base_domain// }" ]] && rm -rf /etc/nginx/certs/${base_domain}
```

Comment:
If prefer that you use [[ -n "${base_domain// }" ]] && ...

Reply:
Sure, that's cleaner. Done.